### ISSUE TITLE
--aggressive flag doesn't work or not implemented?

### ISSUE THREAD

**Issue by danielvanmil** [alvare/discogs2pg#9]
Passing the --aggressive results in an error. I also can't find anything in the source code? Is this implemented?

**Comment by clrnd** [comment]
Yeah it is, it just runs the import on every file at the same time, [it's here](https://github.com/alvare/discogs2pg/blob/master/src-exe/Main.hs#L34).
